src/main/java/teammates/logic/api/EmailGenerator.java

```diff
@@ -456,23 +456,40 @@ private EmailWrapper generateFeedbackSessionEmailBaseForInstructors(
 
         List<EmailWrapper> emails = new ArrayList<EmailWrapper>();
 
+        String instructorFragmentArgumentInstructors = "<p>The email below has been sent to students of course: "
+                + SanitizationHelper.sanitizeForHtml(course.getId()) + ", "
+                + SanitizationHelper.sanitizeForHtml(course.getName())
+                + ".<br>" + Const.EOL + "<br>" + Const.EOL
+                + "=== Email message as seen by the students ===</p>" + Const.EOL;
```

Comment:
@leeyimin now that I have been familiarized with the email issues at this project, in case you have any other issue in mind that is similar to this one, please let me know in order to contribute more. I really want to do so.